server/plugin/api.go

```diff
@@ -246,10 +246,15 @@ func (p *MatterpollPlugin) handleAddOption(vars map[string]string, request *mode
 		return commandErrorGeneric, nil, errors.Errorf("failed to get submission key: %s", addOptionKey)
 	}
 
-	if err = poll.AddAnswerOption(answerOption); err != nil {
+	userLocalizer := p.getUserLocalizer(poll.Creator)
+
+	if errMsg := poll.AddAnswerOption(answerOption); errMsg != nil {
 		response := &model.SubmitDialogResponse{
 			Errors: map[string]string{
-				addOptionKey: err.Error(),
+				addOptionKey: p.LocalizeWithConfig(userLocalizer, &i18n.LocalizeConfig{
```

Comment:
It seems more clearer by creating new func `LocalizeErrorMessageWithConfig(*i18n.Localizer, poll.ErrorMessage) string` that wraps `LocalizeWithConfig`.

Reply:
Good idea :+1: